packages/react/src/components/Popup/Popup.tsx

```diff
@@ -122,62 +123,25 @@ function useRestoreFocus(props: IPopupProps, root: React.RefObject<HTMLDivElemen
 }
 
 function useHideSiblingNodes(props: IPopupProps, root: React.RefObject<HTMLDivElement | undefined>) {
-  const isModalOrPanel = props['aria-modal'];
   // eslint-disable-next-line deprecation/deprecation
-  const enableAriaHiddenSiblings = props.enableAriaHiddenSiblings;
+  const shouldHideSiblings = props['aria-modal'] && props.enableAriaHiddenSiblings;
```

Comment:
We should refine this -- on a pedantic note, checking `aria-modal` isn't a great way to determine whether something is modal, since it's sort-of true by default for certain roles and sort-of the dialog element. With emphasis on sort-of, because the spec is unclear and different browsers/AT handle it differently. At minimum, we'd need to explicitly check that `props['aria-modal'] === 'true'`, since it could also be set to `'false'` (and 
 actually _should_ be explicitly set to false for non-modal dialogs).

I'm not super enthused about adding another prop for modal vs. modeless. Maybe we check `props['aria-modal'] === 'true'` now, and just say that we require it to be declared explicitly for hiding sibling nodes?

Reply:
Per the react typings it could also be `true` (boolean). So I think `String(props['aria-modal']).toLowerCase() === 'true'` would work.